.github/workflows/ci.yml

```diff
@@ -201,6 +201,28 @@ jobs:
       - check-release
     uses: ./.github/workflows/p2p-tests.yml
 
+  stacks-signer-tests:
+    if: |
+      needs.check-release.outputs.is_release == 'true' || (
+        github.event_name == 'workflow_dispatch' ||
+        github.event_name == 'pull_request' ||
+        github.event_name == 'merge_group' ||
+        (
+          contains('
+            refs/heads/master
+            refs/heads/develop
+            refs/heads/next
+          ', github.event.pull_request.head.ref) &&
+          github.event_name == 'push'
+        )
```

Comment:
I don't think this section is needed. We probably just want to run these on PRs and in the merge queue, which the above checks would cover.

Reply:
> I don't think this section is needed. We probably just want to run these on PRs and in the merge queue, which the above checks would cover.

if we do add this as a required status, we kind of need to run this test on any PR/merge action. if we only run it when added to the merge queue, the PR will not be mergeable since this test won't run *until* it's been added to the queue. 

i think combined with my later comment about using the nextest archive, we can run these tests on any commit/merge and the time taken will be relatively short.

Reply:
The section I highlighted here is for running it after it merges into master, develop, or next, which I think can be removed. The section above that will still trigger these on PR action and merge queue.